src/components/CodeView/styles.module.scss

```diff
@@ -0,0 +1,36 @@
+@import '../../scss/variables';
+
+.CodeView {
+  border-radius: $border-radius;
+  border: 1px solid $light-gray;
+  margin-bottom: $default-padding;
+  padding: $default-padding;
+}
+
+.table {
+  display: block;
+  width: 100%;
+}
+
+.tableBody {
+  display: block;
+  overflow: auto;
+  width: 100%;
+}
+
+.lineNumber {
+  padding-right: $default-padding;
+  text-align: right;
+  user-select: none;
+}
+
+.code,
+.lineNumber {
+  font-family: Consolas, Monaco, 'Andale Mono', 'Ubuntu Mono', monospace;
```

Comment:
Can we sync this and other styles with `DiffView`? It could be done in a follow-up, of course.

Reply:
Yep, I think this actually comes from `react-diff-view`. I'll file an issue to sync both.

Reply:
I filed: https://github.com/mozilla/addons-code-manager/issues/298